Algorithm/QCAlgorithm.Trading.cs

```diff
@@ -454,7 +496,7 @@ public List<int> Liquidate(string symbolToLiquidate = "")
                 }
 
                 //Liquidate at market price.
-                orderIdList.Add(Order(symbol, quantity));
+                orderIdList.Add(Order(symbol, quantity).OrderId);
```

Comment:
This could unintentionally add order IDs with a value of zero to this list. First we should check the order response to verify that it is processed:

```C#
                //Liquidate at market price.
                var orderResponse = Order(symbol, quantity);
                if (orderResponse.IsProcessed())
                {
                    orderIdList.Add(orderResponse.OrderId);
                }
```